Gruntfile.js

```diff
@@ -555,6 +555,13 @@ module.exports = function (grunt) {
     ));
   });
 
+  grunt.task.registerTask('generate-readme', 'Generate Readme.md file using package.json', function () {
+    var readmeTpl = grunt.file.read('./README_TPL.md');
+    var version = grunt.file.readJSON('package.json').version;
```

Comment:
probably add a log line about the version being updated:
Something like:
`grunt.log.writeln('Updating version in README.md to ' + version);`